syntax = "proto3";



package CS1_CS2;


service CS1CS2_Greeter{
    // 精确查询
    rpc exactQuery( PreResults)  returns (ExactResult) {}

}
message  track_Sel{
    double  dis = 1;
    int32  lable =2;
}

message PreResults {
   int32 k = 1;
   repeated string ESD = 2;
   repeated double timeList = 3;
}

message ExactResult {
    repeated track_Sel kid =1; //kid
}

